src/preact/bento-ce.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {applyStaticLayout} from '../layout';
+import {initLogConstructor} from '../log';
+import {toWin} from '../types';
+
+/** @type {typeof AMP.BaseElement} */
+let BaseElement;
+
+if (typeof AMP !== 'undefined' && AMP.BaseElement) {
+  BaseElement = AMP.BaseElement;
+} else {
+  class CeBaseElement {
```

Comment:
Did you experiment with trying to only have a CustomElement definition instead of a separate BaseElement as well? Do you think Bento components rely on helpers and utilities that would break without continuing the split